flexget/task.py

```diff
@@ -159,13 +159,13 @@ def __repr__(self) -> str:
         return f'<EntryContainer({list.__repr__(self)})>'
 
 
-class TaskAbort(Exception):
+class TaskAbortError(Exception):
```

Comment:
Don't think this is better. It's not necessarily an error, it can be similar to GeneratorExit, which also doesn't have the Error suffix.